content/en/seavoice/stt/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title : "Speech-to-Text"
+title : "SeaVoice Speech to Text Transcription with Artificial Intelligence"
```

Comment:
"with Deep Learning"?

Reply:
I would personally prefer deep learning as well, but chose Artificial Intelligence for the same reason as above: 90,500 vs. 18,100 (deep learning) in terms of search volume from surfer stats.